transaction.py

```diff
@@ -294,6 +282,21 @@ def cancel_authorize_net(self):
             self.save()
             TransactionLog.serialize_and_create(self, result)
 
+    def get_authorize_net_order_details(self):
```

Comment:
I think its not only sale order could be send. Method should be renamed to `get_authorize_data` or something better if you have